lib/aur-search

```diff
@@ -26,8 +26,8 @@ json_short() {
         read -r OutOfDate
         read -r Description
     }; do
-        [[ "$OutOfDate" == "null" ]] && OutOfDate="" || OutOfDate="(Out-of-date: $(date -d @"$OutOfDate" '+%d %B %Y'))"
-        [[ "$Maintainer" != "null" ]] && Maintainer="" || Maintainer="(Orphaned)"
+        [[ "$OutOfDate" ]] && OutOfDate="(Out-of-date: $(date -d @"$OutOfDate" '+%d %B %Y'))"
+        [[ "$Maintainer" == "" ]] && Maintainer="(Orphaned)" || Maintainer=""
```

Comment:
You could remove this line by using `${Maintainer:-(Orphaned)}` in the `printf` command below. :blush:

Reply:
But if maintainer is not empty, we need to clear its value (not to print their name)

Reply:
Ah, right.

Reply:
This works though: https://github.com/AladW/aurutils/commit/05cb9e36c15e9972e547a7cde0919226ac2498dd

Reply:
Nice trick!